ofl/bodonimodasc/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1 @@
+N/A
```

Comment:
Same comment as in Marc's in Baskervville: It'd be great if we could take the html from the non-SC family and simply add a line mentioning it's the Small Cap variant and provide a link back to the non-SC family. This can probably be place just before the "To contribute" part.